storybookjs/storybook
# WIP: Aurelia 2 Support

Issue:

## What I did
Initial Implementation of Aurelia 2 for storybooks.
## How to test

- Is this testable with Jest or Chromatic screenshots? No.
- Does this need a new example in the kitchen sink apps? Done.
- Does this need an update to the documentation? Included.

If your answer is yes to any of these, please make sure to include it in your PR.
